### DISCUSSION TITLE
2005/11/21/%E7%94%A8wix%E7%BC%96%E5%86%99msi%E5%AE%89%E8%A3%85%E5%8C%85

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#61] [Announcements]
# 2005/11/21/%E7%94%A8wix%E7%BC%96%E5%86%99msi%E5%AE%89%E8%A3%85%E5%8C%85

WiX（Windows Installer XML）是Rob Mensching（Windows组的一个SDE）写的一个用来编写MSI安装包的工具集，已经被微软内部的很多产品组采用，例如Office组，以及我所在的MSN组，等等。

https://blog.qizhen.xyz/2005/11/21/%E7%94%A8wix%E7%BC%96%E5%86%99msi%E5%AE%89%E8%A3%85%E5%8C%85

**Comment by ruanqizhen** [comment]
之前博客下的留言：

亮
2005年12月15日 09:09 编辑
很好很好，找到个空间总算有点意义的！哈哈

回复
Qizhen
2005年12月11日 15:25 编辑
这篇文章不是我写的，是我偶尔看到的，正好我也要用 wix 就把它放到这了。

回复
Emerson
2005年12月10日 23:49 编辑
是您本人的文章么？？
